lib/tensorflow/graph.rb

```diff
@@ -96,11 +96,11 @@ def make_attr_value(attribute_type, value)
   end
 
   TYPE2ENUM = {
-    DT_FLOAT: Tensorflow::TF_FLOAT,
-    DT_DOUBLE: Tensorflow::TF_DOUBLE,
-    DT_INT64: Tensorflow::TF_INT64,
-    DT_STRING: Tensorflow::TF_STRING,
-    DT_COMPLEX128: Tensorflow::TF_COMPLEX128
+    DT_FLOAT: Tensorflow::Internal::TF_FLOAT,
+    DT_DOUBLE: Tensorflow::Internal::TF_DOUBLE,
+    DT_INT64: Tensorflow::Internal::TF_INT64,
+    DT_STRING: Tensorflow::Internal::TF_STRING,
+    DT_COMPLEX128: Tensorflow::Internal::TF_COMPLEX128
```

Comment:
Yes @zverok. This is bit confusing because the enum is actually defined at in [tensor_c_api.h](https://github.com/Arafatk/tensorflow.rb/blob/master/ext/sciruby/tensorflow_c/files/tensor_c_api.h#L73)  and are included under swig [module](https://github.com/Arafatk/tensorflow.rb/blob/master/ext/sciruby/tensorflow_c/Tensorflow.i#L24).